releases.yaml

```diff
@@ -8,6 +8,7 @@ lts:
   slug: FocalFossa
   short_version: "20.04"
   full_version: "20.04.2"
+  full_version_desktop: "20.04.2.0"
```

Comment:
Is this not supposed to be 20.04.3.0?

Reply:
I wish but nope, it's a fix for a broken release: https://github.com/canonical-web-and-design/ubuntu.com/blob/master/releases.yaml#L14